src/main/java/functional/algebraic/Maybe.java

```diff
@@ -62,6 +62,7 @@
  *
  * @param <V> The type of the optional value
  * @author Zoey Hewll
+ * @author Eleanor McMurtry (renamed match overloads to avoid inference errors)
```

Comment:
Remove comment on why you're listed as an author. I'm happy to share credit, and people can check the blame for specific feature authorship.
```suggestion
 * @author Eleanor McMurtry
```